chain/chain/src/stateless_validation/chunk_validation.rs

```diff
@@ -157,12 +173,27 @@ pub fn pre_validate_chunk_state_witness(
                 break;
             }
         }
+        (last_chunk_shard_id, last_chunk_shard_index)
+    };
+
+    let last_chunk_block = blocks_after_last_last_chunk.first().ok_or_else(|| {
```

Comment:
sanity check for myself 
- the blocks in blocks_after_last_last_chunk are in reverse chronological order 
- the blocks include the last chunk block (it's inclusive)